hypervisor/qemu/qemu.go

```diff
@@ -35,17 +34,6 @@ func qemuContext(ctx *hypervisor.VmContext) *QemuContext {
 	return ctx.DCtx.(*QemuContext)
 }
 
-func InitDriver() *QemuDriver {
-	cmd, err := exec.LookPath("qemu-system-x86_64")
```

Comment:
I prefer the second one.
thanks and it is very nice for you to provide both ways.